.github/workflows/tests.yml

```diff
@@ -25,3 +25,12 @@ jobs:
                   go test -race -cover ./...
               env:
                   GOPATH: /home/runner/work/gocity/go
+            - name: Generate coverage report
+              run: |
+                    go test `go list ./...` -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
The go test command is duplicated. Can you only include the parameters `-coverprofile=coverage.txt -covermode=atomic.` in the previous step?

Reply:
Yep, on my way.

Reply:
Done.